small text fixes to security-authentication-claims doc (#22846)
dotnet/AspNetCore.Docs: aspnetcore/security/authentication/claims.md

* Reset the claims namespaces
* Customize, extend the claims using <xref:Microsoft.AspNetCore.Authentication.IClaimsTransformation.TransformAsync%2A>

## Mapping claims using OpenID Connect authentication

> The profile claims can be returned in the `id_token` which is returned after a successful authentication. The ASP.NET Core client application just needs to request the profile scope. When using the `id_token` for claims, no extra claims mapping is required.

Reviewer: More nits here. Try ...... (with "app" ... that's ok) ...

```suggestion
The profile claims can be returned in the `id_token`, which is returned after a successful authentication. The ASP.NET Core client app only requires the profile scope. When using the `id_token` for claims, no extra claims mapping is required.
```

Author: thanks for the review, updated with you improvements

## Claims namespaces, default namespaces

> ASP.NET Core adds default namespaces to some known claims which might not be required in your application. You can disable this namespace adding if you would like to use the claims exactly like the Open ID Connect server created. 

Reviewer: Routinely, we use "app" everywhere, so that bit is ok. However, the "you/your" bits and some of the other phrasing could use a touch, and there's a missing comma for the non-essential clause.

```suggestion
ASP.NET Core adds default namespaces to some known claims, which might not be required in the app. Optionally, disable these added namespaces and use the exact claims that the OpenID Connect server created.
```

... something along those lines.

Author: thanks for the review, updated with you improvements
